pyqtgraph/parametertree/Parameter.py

```diff
@@ -219,7 +219,6 @@ def __init__(self, **opts):
             self.setValue(value)
 
         if 'default' not in self.opts:
-            self.opts['default'] = None
```

Comment:
I think I'm -1 on this change here (combined with using `self.opts.get('default')`.  `dict.get()` is useful, but it can mask other bugs in code, and raising KeyError exceptions may contribute to "failing fast", which should significantly simplify debugging efforts.

Reply:
Thanks for your thoughts. Well, I see your point on "failing fast". However, here I'd rate it less than clearness of code and the promise of `setDefault()`: _set the default_.
As is, calling `setDefault()` requires making sure there's already some "default" (and which gets replaced), otherwise you risk a `KeyError` (which was actually my problem when calling it from my code).
In general, I think interface methods should do the complete job, in order to avoid the need of accessing the underlying data structure `opts` and using the correct key `default`, from outside again and again.